docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md

```diff
@@ -37,20 +35,20 @@ Feature Support Matrix
 
 The table below demonstrates support of key features by OpenVINO device plugins.
 
- ========================================================================================= =============== =============== =============== ======================== 
-  Capability                                                                                CPU             GPU             GNA             Arm® CPU  
- ========================================================================================= =============== =============== =============== ======================== 
-  :doc:`Heterogeneous execution <openvino_docs_OV_UG_Hetero_execution>`                     Yes             Yes             No              Yes                     
-  :doc:`Multi-device execution <openvino_docs_OV_UG_Running_on_multiple_devices>`           Yes             Yes             Partial         Yes                     
-  :doc:`Automatic batching <openvino_docs_OV_UG_Automatic_Batching>`                        No              Yes             No              No                      
-  :doc:`Multi-stream execution <openvino_docs_deployment_optimization_guide_tput>`          Yes             Yes             No              Yes                     
-  :doc:`Models caching <openvino_docs_OV_UG_Model_caching_overview>`                        Yes             Partial         Yes             No                      
-  :doc:`Dynamic shapes <openvino_docs_OV_UG_DynamicShapes>`                                 Yes             Partial         No              No                      
-  :doc:`Import/Export <openvino_inference_engine_tools_compile_tool_README>`                Yes             No              Yes             No                      
-  :doc:`Preprocessing acceleration <openvino_docs_OV_UG_Preprocessing_Overview>`            Yes             Yes             No              Partial                 
```

Comment:
do we need to drop column for ARM CPU?
Do we have scalar kernels for preprocessing? if no, preprocessing is not supported on ARM

Reply:
Fixed